velox/type/tests/TimestampTest.cpp

```diff
@@ -418,5 +418,71 @@ TEST(TimestampTest, tmToStringTimestamp) {
   // %FT%T - equivalent to "%Y-%m-%dT%H:%M:%S" (the ISO 8601 timestamp format)
   testTmToString("%FT%T", TimestampToStringOptions::Mode::kFull);
 }
+
+TEST(TimestampTest, leadingPositiveSign) {
+  TimestampToStringOptions options = {
+      .leadingPositiveSign = true,
+      .zeroPaddingYear = true,
+      .dateTimeSeparator = ' ',
+  };
+  auto testTmToString = [&](Timestamp ts, const std::string& expected) {
```

Comment:
This lambda repeats multiple times. Consider extracting it into a helper method.